lib/api/useApiFetch.tsx

```diff
@@ -99,9 +100,16 @@ export default function useApiFetch() {
       const shardsIds = Object.keys(shards);
 
       response = shardsIds.reduce((acc, shardId) => {
-        const shardResponse = (response as Record<ShardId, {data: SuccessType}>)[shardId]['data'] || [];
-        if (shardResponse && Array.isArray(shardResponse) && shardResponse.length > 0) {
-          acc.push(...shardResponse as Array<never>);
+        const shardResponse = (response as Record<ShardId, {data: SuccessType}>)[`s${ shardId }`]['data'] || [];
```

Comment:
`shardId` field from multisearch proxy can be also configured for using new id for shards, so we no need add `s` prefix